src/SourceBuild/content/repo-projects/installer.proj

```diff
@@ -148,4 +148,6 @@
           UseHardlinksIfPossible="true" />
   </Target>
 
+  <Import Project="$(RepositoryEngineeringDir)sdkArchiveDiff.targets" Condition="'$(PortableBuild)' == 'true' and '$(PgoInstrument)' != 'true'" />
```

Comment:
Should this also be conditioned on source-only builds?

Reply:
You mean on non-source-only builds, right? That's already handled via the PortableBuild condition as source-builds are by definition non-portable.

Reply:
Sure